lib/devcycle-ruby-server-sdk/localbucketing/events_payload.rb

```diff
@@ -0,0 +1,32 @@
+require 'sorbet-runtime'
+
+module DevCycle
+  class EventsPayload
+    extend T::Sig
+
+    attr_accessor :records
```

Comment:
be careful with this. There was a bug in Go where it was just sending this data format verbatim to the events API (the API expects a "batch" field not "records"

Reply:
yeah - this is just the deserialized output from the wasm; this would have to be transferred to an HTTP post